security-malware-firewall.php

```diff
@@ -809,6 +809,8 @@ function spbc_authenticate($user, $username, $password){
 	
 	if(is_wp_error($user)){
 		
+		$spbc->login_error = true;
```

Comment:
adding a new field to an object just on the fly?

Reply:
Yep) Since we already have a global variable $spbc, why don't use it?